programs/ssl/ssl_client2.c

```diff
@@ -347,8 +347,9 @@ int main(void)
 
 #if defined(MBEDTLS_SSL_EARLY_DATA)
 #define USAGE_EARLY_DATA \
-    "    early_data=%%d        default: 0 (disabled)\n"      \
-    "                        options: 0 (disabled), 1 (enabled)\n"
+    "    early_data=%%s      The file path to read early data from\n" \
```

Comment:
Then parameter name should align with internal variable name.

Reply:
Keep the input option as this name, just to align with the name from gnutls and openssl.

Reply:
prefer change it to `early_data_file` , same with `opt.early_data_file`

Reply:
See the option of GNUTLS, `--earlydata`, and the option of OpenSSL, `"-early_data"`. Both of them expect one input file but didn't name it as early_data_file. Why not keep aligned with them?
We will provide the comment to expect one input file, and internally stored it into one variable named xxx_file to facilitate the maintaining.

Reply:
Or change `opt.early_data_file` to `opt.early_data` , up to you. But that should be same